api/proto/teleport/legacy/client/proto/authservice.proto

```diff
@@ -2053,12 +2053,26 @@ message PaginatedResource {
   // that IncludeRequestable.
   bool RequiresRequest = 14 [(gogoproto.jsontag) = "requires_request,omitempty"];
 
+  // TargetHealthStatuses is a list of health statuses for the resource. There
+  // can be multiple statuses if multiple agents proxy the same resource.
+  repeated TargetHealthStatus TargetHealthStatuses = 17 [(gogoproto.jsontag) = "target_health_statuses,omitempty"];
```

Comment:
alternatively maybe use a message similar to `types.TargetHealthGroups` but only counts? like `enriched_resource.target_counts_by_health_status.healthy = 5` (i am not at naming, just an example)

Reply:
the count struct/message is more deterministic than a list that can be any size like `[0,0,1,1,0,0,1,1,2,1,2,1,1,1,1,0]` and the list requires enum conversion. (i know this example is unrealistic but you never know =p)